src/view/wlr-surface-node.cpp

```diff
@@ -486,6 +487,7 @@ void wf::scene::wlr_surface_node_t::update_pending_outputs()
             {
                 wlr_surface_send_enter(surface, wo->handle);
                 wlr_fractional_scale_v1_notify_scale(surface, wo->handle->scale);
```

Comment:
Maybe we should move this below as well, so that we only notify the highest fractional scale?